cfy_manager/components/postgresql_server/postgresql_server.py

```diff
@@ -483,17 +506,28 @@ def _configure_cluster(self):
         fd, tmp_path = mkstemp()
         os.close(fd)
         with open(tmp_path, 'w') as pg_conf:
```

Comment:
plz move the `with open` under the big dict, so that it's closer to the actual write and the `with` scope is smaller